netbench/netbench-driver/src/bin/netbench-driver-s2n-quic-client.rs

```diff
@@ -58,6 +59,11 @@ impl Client {
 
         let mut io_builder =
             io::Default::builder().with_receive_address((self.opts.local_ip, 0u16).into())?;
+        
+        match env::var("DISABLE_GSO") {
```

Comment:
You don't need this if you just use structopt

Reply:
Cool, I'll edit that out